timesketch/ui/graphs/cytoscape.component.spec.ts

```diff
@@ -0,0 +1,276 @@
+import {Component, ViewChild} from '@angular/core'
+import {TestBed, ComponentFixture} from '@angular/core/testing'
+import {BrowserDynamicTestingModule, platformBrowserDynamicTesting} from '@angular/platform-browser-dynamic/testing'
+import {BrowserModule} from '@angular/platform-browser'
+
+import {CytoscapeComponent} from './cytoscape.component'
+
+export const testCytoscapeOptions = Object.seal({
+  zoomingEnabled: false,
+  userZoomingEnabled: false,
+  panningEnabled: false,
+  userPanningEnabled: false,
+  boxSelectionEnabled: true,
+  autolock: true,
+  autoungrabify: true,
+  autounselectify: true,
+  headless: true,
+  hideEdgesOnViewport: true,
+  hideLabelsOnViewport: true,
+  textureOnViewport: true,
+  motionBlur: true,
+  styleEnabled: false,
+  minZoom: 1e-10,
+  maxZoom: 1e10,
+  touchTapThreshold: 10,
+  desktopTapThreshold: 6,
+  motionBlurOpacity: 0.4,
+  wheelSensitivity: 0.8,
+  pixelRatio: 1.0,
+  selectionType: 'additive',
+})
+
+export function getOptionFromCytoscape(_cy: Cy.Core, option: string) {
+  type CytoscapeInternals = {
```

Comment:
All of the "any"s in the type definition can be replaced with {}.

Reply:
Done.